lumen/ai/coordinator.py

```diff
@@ -640,19 +640,6 @@ async def _resolve_plan(self, plan, agents, messages) -> tuple[list[ExecutionNod
                     render_output=step.render_output
                 )
             )
-            if "current_table" in unmet_dependencies:
-                unmet_dependencies.remove('current_table')
-                if not table_provided:
-                    execution_graph.append(
-                        ExecutionNode(
-                            agent=agents['SQLAgent'],
-                            provides=['current_table'],
-                            instruction='Load the table',
-                            title='Loading table',
-                            render_output=False
-                        )
-                    )
-                table_provided = True
```

Comment:
I forget the exact reason of adding this, but it was calling SQLAgent twice in a row which isn't quite correct.
```
<AnalystAgent AnalystAgent02022>
<SQLAgent SQLAgent02024>
<SQLAgent SQLAgent02024>
```

I haven't encountered any errors since testing without it.

Reply:
There's good reason to add it, but it should indeed not add it twice.

Reply:
I think this was only necessary because there was a bug in resolving dependencies, but was fixed in: https://github.com/holoviz/lumen/pull/761?

Reply:
I don't think that's accurate, the whole point of that fix was to move this block to the correct place. Let's undo this and resolve this problem separately.

Reply:
Okay, inverted the resolution now, which should avoid the problem.

Reply:
It wasn't super easy to trigger the problem that required this change but it was possible. You also always force it by asking it to skip the SQL agent.